### PR TITLE
feat: Added Login Page

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
- Login Page from N-store.

### CODE REVIEW

**Review by aryan-13 — APPROVED** [review]
The code looks clean and well-formatted, looks good to merge!

**Review by anshikaa12 — APPROVED** [review]
code is clean and readable.Good to merge!

File: Login/login.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+.login-container {
+    display: flex;
+    flex-direction: row;
+    background-color: var( --secondary-bg-color);
+    padding: 10rem;
+    border-radius: 1rem;
+}
+
+.herostore-title {
+    font-size: xx-large;
```
Comment (omjadhav85) [inline Login/login.css]: better to use variable here

File: Login/login.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                        <img src="/Assets/Images/Ninja.png" alt="Ninja" srcset="">
+                    </div>
+
+                    <div class="login-form flex-center">
+                        <!--  -->
+                        <div class="form-container">
+                            <h2 class="header">Login</h2>
+                            <form class="form" id="form">
+
+
```
Comment (omjadhav85) [inline Login/login.html]: remove extra spaces

File: Login/login.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply Login/login.css]: Thanks, will do that.

**Comment by omjadhav85** [comment]
Just some basic suggestions...else great work!!

**Pull request merged**